src/platform/forms-system/src/js/review/SubmitController.jsx

```diff
@@ -122,14 +126,21 @@ class SubmitController extends Component {
   };
 
   render() {
-    const { form, formConfig } = this.props;
+    const {
+      form,
+      formConfig,
+      renderErrorMessage,
+      formErrors = {},
+    } = this.props;
 
     return (
       <SubmitButtons
         formConfig={formConfig}
         onBack={this.goBack}
         onSubmit={this.handleSubmit}
         submission={form.submission}
+        renderErrorMessage={renderErrorMessage}
```

Comment:
This isn't used by `SubmitButtons`. Should it be? I don't see it passed to this component. :confused:

Reply:
You're right! I'll remove it. I think it's a remanent from a time before the alerts were split out.